<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accounts</class>
 <widget class="QWidget" name="Accounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="AccountsView" name="accountsView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QCheckBox" name="passwordsCheckBox">
         <property name="text">
          <string>store passwords on disk</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="disclaimerButton">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="icon">
          <iconset resource="../../res/resources.qrc">
           <normaloff>:/icons/warning_16.png</normaloff>:/icons/warning_16.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QToolButton" name="addAccountButton">
       <property name="icon">
        <iconset resource="../../res/resources.qrc">
         <normaloff>:/icons/add_48.png</normaloff>:/icons/add_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteAccountButton">
       <property name="icon">
        <iconset resource="../../res/resources.qrc">
         <normaloff>:/icons/cancel_48.png</normaloff>:/icons/cancel_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountsView</class>
   <extends>QTreeView</extends>
   <header>accountsview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>accountsView</tabstop>
  <tabstop>addAccountButton</tabstop>
  <tabstop>deleteAccountButton</tabstop>
  <tabstop>passwordsCheckBox</tabstop>
  <tabstop>disclaimerButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
